package.json

```diff
@@ -2,7 +2,7 @@
   "name": "blockstack",
   "version": "0.3.10",
   "description": "The Blockstack Javascript library for identity and authentication.",
-  "main": "lib/index",
+  "main": "dist/blockstack.js",
```

Comment:
This is the browserified version intended for apps using this package in the browser in an app that doesn't have its own browserify/webpack system. The previous entry point allows package users to use this on node or in the browser via their preferred browserify/webpack system. Does this make sense?